.eslintrc

```diff
@@ -10,6 +10,7 @@
   },
   "rules": {
     "func-names": "off",
+    "react/destructuring-assignment": "off",
```

Comment:
I would say that this is the most stupid ESlint rule I've ever seen. https://eslint.org/docs/rules/prefer-destructuring is OK, but this? Its requirements do not make sense to me so I've turned it off. Also it detects only some lines and some do not.